packages/saber-plugin-query-posts/lib/index.js

```diff
@@ -49,10 +49,17 @@ exports.apply = (api, options = {}) => {
         const pagePublicFields = api.pages.getPagePublicFields(page)
         allPosts.add(pagePublicFields)
 
+        page.tags = []
+        page.categories = []
+
         // Group posts for tag pages
         const tags = [].concat(page.attributes.tags || [])
         if (tags.length > 0) {
           for (const tag of tags) {
+            page.tags.push({
+              name: tag,
+              link: permalinks.tag.replace(/:name/, tag)
```

Comment:
Why not use `permalink`.

Reply:
@h404bi I am not sure if I understand what you mean, but by any chance do you mean that I should change `link: permalinks.tag.replace(/:name/, tag)` to `permalink: permalinks.tag.replace(/:name/, tag)`?

Reply:
I just realized that it recommend to use `{name, link}` in https://github.com/egoist/saber/projects/1#card-20259259 , So never mind.

Reply:
Actually `permalink` makes more sense 😆